lib/instrumentald/server_controller.rb

```diff
@@ -179,10 +198,25 @@ def telegraf_template_config_path
     end
   end
 
+  def mongodb_configs
+    mongodb_servers = Array(config_file["mongodb"])
+
+    mongodb_configs = mongodb_servers.map do |uri|
+      begin
+        u = Mongo::URI.new(uri)
+        u.servers.map do |server|
```

Comment:
I think it would be better if each mongo connection string in the instrumentald config mapped to one config block in the telegraf config file, to keep servers grouped logically in the telegraf plugin.

So, the output of this block would be the same strings being output now, but in one array in one hash.  In the 3-string example, we'd end up with 3 hashes, one of which contained 3 servers.

something like
```ruby
  def mongodb_configs
    mongodb_connection_strings = Array(config_file["mongodb"])

    mongodb_configs = mongodb_connection_strings.map do |uri|
      begin
        u = Mongo::URI.new(uri)
        servers = u.servers.map do |server|
          creds = "#{u.credentials[:user]}:#{u.credentials[:password]}@" if u.credentials.values.any?
          "mongodb://#{creds}#{server}"
        end
        {:servers => servers, :ssl => u.uri_options[:ssl]}
      rescue Mongo::Error::InvalidURI => ex
        {:servers => [uri]}
      end
    end.flatten.compact
end
```